src/pages/workspaces/workspace/analysis/AnalysisLauncher.js

```diff
@@ -567,6 +576,7 @@ const AnalysisEditorFrame = ({
   const [busy, setBusy] = useState(false)
   const [analysisSetupComplete, setAnalysisSetupComplete] = useState(false)
   const cookieReady = useStore(cookieReadyStore)
+  const cloudPlatform = !!googleProject ? cloudProviderTypes.GCP : cloudProviderTypes.AZURE
```

Comment:
naive js question on my part, but why does this constant has to be defined the same way again? is there a DRY way to set this globally?

Reply:
its in a different functional component, so even if its in the same file they have different scopes. Realistically, the workspace store should have the `cloudPlatform` information, but short of that we should use the function `getCloudProviderFromWorkspace` in my opinion

Reply:
Yeah. We could use some updated data models.